components/ReviewSection.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Heading,
+  HeadingLevels,
+  Paragraph,
+  SummaryList,
+} from "lbh-frontend-react";
+import React from "react";
+import PropTypes from "../helpers/PropTypes";
+import { SectionRow } from "../helpers/useReviewSectionRows";
+
+interface Props {
+  section?: {
+    heading?: string;
+    rows?: SectionRow[];
+  };
+}
```

Comment:
```suggestion
interface Props {
    heading?: string;
    rows?: SectionRow[];
}
```

This would be more idiomatic, I think.